sandbox/plc4c/spi/include/plc4c/spi/types_private.h

```diff
@@ -178,7 +198,20 @@ struct plc4c_request_value_item_t {
 struct plc4c_response_value_item_t {
   plc4c_item *item;
   plc4c_response_code response_code;
-  void *value;
+  plc4c_data *value;
+};
+
```

Comment:
This maybe where we want to have the 'cookie'

Reply:
Don't understand this cookie thing ...

Reply:
I can't find an example, so let's forget it.
It is basically that you would want to store come context item or information in the request item, so that when you got the callback you could identify the object and use the context instead of having to map the request item to something else elsewhere